mod/activity/views/default/river/filter.php

```diff
@@ -10,7 +10,7 @@
 	return;
 }
 
-elgg_require_js('river/filter');
+elgg_require_js('./river/filter.mjs');
```

Comment:
Probably want a separate function like `elgg_import('./river/filter.mjs')`. I think it would be important to use the same URI format as we would in ES modules.